templates/home/index.html.twig

```diff
@@ -0,0 +1,3 @@
+ <a href="{{ path('app_contact') }}">Contact</a>
```

Comment:
This file needs to be a html file that extends the base.html.twig class.
https://twig.symfony.com/doc/2.x/tags/extends.html

In this template we will display dynamically links sent from the controller as an array.